app/component/MobileView.js

```diff
@@ -4,7 +4,9 @@ import { configShape } from '../util/shapes';
 import MapBottomsheetContext from './map/MapBottomsheetContext';
 import MobileFooter from './MobileFooter';
 
-function slowlyScrollTo(el, to = 0, duration = 1000) {
+const BOTTOM_SHEET_OFFSET = 20;
+
+function slowlyScrollTo(el, to = BOTTOM_SHEET_OFFSET, duration = 1000) {
```

Comment:
Is this change 0 -> 20 intentional?

Reply:
yes, otherwise navi buttons wil not show properly.